package.json

```diff
@@ -35,7 +35,8 @@
     "rollup": "^1.0.0"
   },
   "dependencies": {
-    "rollup-pluginutils": "^2.3.3"
+    "rollup-pluginutils": "^2.3.3",
+    "fs-extra": "^9.0.0"
```

Comment:
Can you move this to devDependencies?

Reply:
This was a dependency, but I moved it just in case: https://github.com/thgh/rollup-plugin-css-only/pull/19/commits/9d1e261849af043c5d828d683a3c7522a8885ff5